pkg/kubelet/apis/config/types.go

```diff
@@ -164,6 +164,8 @@ type KubeletConfiguration struct {
 	HealthzPort int32
 	// healthzBindAddress is the IP address for the healthz server to serve on
 	HealthzBindAddress string
+	// HealthzCRI enables cri check for the healthz server.
+	HealthzCRI bool
```

Comment:
There's a few other places you will need to add this if you want to add it as a kubelet config option (see https://github.com/kubernetes/kubernetes/pull/108941 as example)

Reply:
Done and tested. Thanks.